src/api/auth/auth.controller.ts

```diff
@@ -4,18 +4,27 @@
 import { AuthService } from './auth.service';
 import { LoginDto } from './dto/login.dto';
 
+
 @ApiTags('Auth')
 @Controller('auth')
 export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'hre' is defined but never used.
<!-- __reviewdog__:ChAyZDI1NDBhNTU4MzU5MjdlEgZlc2xpbnQ= -->